website/src/pages/Community/index.tsx

```diff
@@ -130,45 +135,54 @@ const Community: React.FC = () => {
-        <div className={classes.sponsorRoot}>
-          <Grid container justify="space-evenly" alignItems="center">
-            <Grid item xs={12} md={4}>
-              <Typography className={classes.sponsorDescription}>
-                {t("community.sponsor.mainSponsor")}
-              </Typography>
-              <img
-                src="../Images/logos/mayadata_logo.svg"
-                alt={t("newsletter.email")}
-                className={classes.sponsorCompany}
-              />
+        <div className={classes.sectionDiv}>
+          <Grid container justify="space-evenly" className={classes.sponsorWrapper}>
+            <Grid item xs={12} md={5}>
+              <div className={classes.sponsorDiv}>
+                <Typography className={classes.sponsorDescription}>
+                  {t("community.sponsor.mainSponsor")}
+                </Typography>
+                <img
+                  src="../Images/logos/mayadata_logo.svg"
+                  alt={t("generic.mayadata")}
+                  className={classes.mayaDataLogo}
+                />
+              </div>
+              
             </Grid>
-            <Grid item xs={12} md={8}>
+            <Grid item xs={12} md={7}>
+            <div className={classes.sponsorDiv}>
               <Typography className={classes.sponsorDescription}>
                 {t("community.sponsor.dependentProjects")}
               </Typography>
-              <img
-                src="../Images/logos/rancher.svg"
-                alt={t("newsletter.email")}
-                className={classes.sponsorCompany}
-              />
-              <img
-                src="../Images/logos/intel.svg"
-                alt={t("newsletter.email")}
-                className={classes.sponsorCompany}
-              />
-              <img
-                src="../Images/logos/gostor.svg"
-                alt={t("newsletter.email")}
-                className={classes.sponsorCompany}
-              />
-              <img
-                src="../Images/logos/openzfs.svg"
-                alt={t("newsletter.email")}
-                className={classes.sponsorCompany}
-              />
+              <div>
+                <img
+                  src="../Images/logos/rancher.svg"
+                  alt={t("generic.rancher")}
+                  className={[classes.sponsorCompany, classes.rancher].join(' ')}
+                />
+                <img
+                  src="../Images/logos/intel.svg"
+                  alt={t("generic.intel")}
+                  className={classes.sponsorCompany}
+                />
+                <img
+                  src="../Images/logos/gostor.svg"
+                  alt={t("generic.gostor")}
+                  className={classes.sponsorCompany}
+                />
+                <img
+                  src="../Images/logos/openzfs.svg"
+                  alt={t("generic.openzfs")}
+                  className={classes.sponsorCompany}
+                />
```

Comment:
this can taken up as a refactor, get the sponsor list from a json

Reply:
Done